src/api/cvc4cpp.cpp

```diff
@@ -5678,6 +5678,74 @@ SmtEngine* Solver::getSmtEngine(void) const { return d_smtEngine.get(); }
  */
 Options& Solver::getOptions(void) { return d_smtEngine->getOptions(); }
 
+/* -------------------------------------------------------------------------- */
+/* Optimization                                                               */
+/* -------------------------------------------------------------------------- */
+
+Objective:: Objective(Term t, ObjectiveType type){
```

Comment:
```suggestion
Objective:: Objective(Term t, ObjectiveType type) {
```

Did you do clang-format?